lib/node_modules/@stdlib/ndarray/base/mskfilter/lib/10d_accessors.js

```diff
@@ -0,0 +1,303 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/* eslint-disable max-depth, max-len */
+
+'use strict';
+
+// MAIN //
+
+/**
+* Applies a mask to a provided input ndarray and assigns unmasked values to elements in a provided one-dimensional output ndarray.
+*
+* @private
+* @param {Object} x - object containing input ndarray meta data
+* @param {Object} mask - object containing mask ndarray meta data
+* @param {Object} y - object containing output ndarray meta data
+* @returns {void}
+*
+* @example
+* var BooleanArray = require( '@stdlib/array/bool' );
+* var Complex64Array = require( '@stdlib/array/complex64' );
+* var Complex64 = require( '@stdlib/complex/float32/ctor' );
+* var realf = require( '@stdlib/complex/realf' );
+* var imagf = require( '@stdlib/complex/imagf' );
+*
+* // Define the shape of the array:
+* var shape = [ 1, 1, 1, 1, 1, 1, 1, 1, 2, 2 ];
+*
+* // Define the array strides:
+* var sx = [ 2, 2, 2, 2, 2, 2, 2, 2, 2, 1 ];
```

Comment:
The strides are incorrect. Should be `[ 4, 4, 4, 4, 4, 4, 4, 4, 2, 1 ]`. See for yourself by using `ndarray/base/shape2strides`.

Reply:
I referred https://github.com/stdlib-js/stdlib/blob/e9c6d40b6d4d382f39d2000c1996bd7ccb38bb29/lib/node_modules/%40stdlib/ndarray/base/assign/lib/10d_accessors.js#L61 this one. Hence, I am using the above strides.

Reply:
Yeah, that example is incorrect and needs to be fixed.